src/test/java/org/example/MyStepDefs.java

```diff
@@ -0,0 +1,48 @@
+package org.example;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+public class MyStepDefs {
+
+    HomePage homePage = new HomePage();
+    RegistrationPage registrationPage = new RegistrationPage();
+    RegistrationSuccessfully registrationSuccessfully = new RegistrationSuccessfully();
+
+    @Given("^user is on HomePage$")
+    public void user_is_on_HomePage()
+    {
+        homePage.toVerifyUserIsOnHomePage();
+
```

Comment:
remove empty line

Reply:
There is curly bracket . So, i can check, i am not missing any curly bracket.